packages/gui/components/Select/Dropdown.tsx

```diff
@@ -28,26 +41,74 @@ const Dropdown: FC<Props> = ({ config }) => {
 	};
 
 	return (
-		<View>
-			<Hoverable style={mutualStyles.button} onPress={handleMainButtonPress}>
+		<View style={styles.container}>
+			<Hoverable
+				style={[mutualStyles.button, mutualStyles.focus]}
+				onPress={handleMainButtonPress}
+			>
 				<Text style={mutualStyles.text}>{title}</Text>
 			</Hoverable>
-			<View>
-				{items.map((item, index) => {
+
+			<View
+				style={[
+					mutualStyles.button,
+					mutualStyles.focus,
+					styles.dropdownContainer,
+				]}
+			>
+				<View style={[mutualStyles.button, mutualStyles.focus, styles.input]}>
+					<TextInput
+						placeholder="Search"
+						onChangeText={(text) => setSearchText(text)}
```

Comment:
use lodash's debounce here: e.g `onChangeText={debouncedChangeText}`

Reply:
could be `lodash throttle` instead of `lodash debounce`, the idea is to limit too frequent function calls in short period of time.